### ISSUE TITLE
build failing on rpi3 running nodejs v13

### ISSUE THREAD

**Issue by grkblood13** [mkrufky/node-dvbtee#38]
[alarm@alarmpi local-channels]$ node -v
v13.12.0
[alarm@alarmpi local-channels]$ uname -a
Linux alarmpi 5.4.18-udlv7+ #1 SMP Sun Feb 16 10:11:01 CST 2020 armv7l GNU/Linux

$ npm install dvbtee

output:

```
  CXX(target) Release/obj.target/dvbtee/src/node-dvbtee.o
  CXX(target) Release/obj.target/dvbtee/src/dvbtee-parser.o
In file included from ../src/dvbtee-parser.cc:11:
../../nan/nan.h: In function 'void Nan::AsyncQueueWorker(Nan::AsyncWorker*)':
../../nan/nan.h:2298:62: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
 2298 |     , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
      |                                                              ^
In file included from ../src/node-dvbtee.cc:11:
../../nan/nan.h: In function 'void Nan::AsyncQueueWorker(Nan::AsyncWorker*)':
../../nan/nan.h:2298:62: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
 2298 |     , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
      |                                                              ^
../src/dvbtee-parser.cc: In static member function 'static void dvbteeParser::Init(Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE)':
../src/dvbtee-parser.cc:44:61: error: no matching function for call to 'v8::Object::Set(v8::Local<v8::String>, v8::Local<v8::Function>&)'
   44 |   exports->Set(Nan::New("Parser").ToLocalChecked(), function);
      |                                                             ^
In file included from /home/alarm/.cache/node-gyp/13.12.0/include/node/node.h:67,
                 from ../../nan/nan.h:54,
                 from ../src/dvbtee-parser.cc:11:
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
 3547 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note:   candidate expects 3 arguments, 2 provided
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
 3550 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note:   candidate expects 3 arguments, 2 provided
../src/node-dvbtee.cc: In function 'void libVersion(const Nan::FunctionCallbackInfo<v8::Value>&)':
../src/node-dvbtee.cc:18:60: error: no matching function for call to 'v8::Array::Set(int, Nan::imp::FactoryBase<v8::Number>::return_t)'
   18 |   version->Set(0, Nan::New<v8::Number>(LIBDVBTEE_VERSION_A));
      |                                                            ^
In file included from /home/alarm/.cache/node-gyp/13.12.0/include/node/node.h:67,
                 from ../../nan/nan.h:54,
                 from ../src/node-dvbtee.cc:11:
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
 3547 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note:   candidate expects 3 arguments, 2 provided
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
 3550 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note:   candidate expects 3 arguments, 2 provided
../src/node-dvbtee.cc:19:60: error: no matching function for call to 'v8::Array::Set(int, Nan::imp::FactoryBase<v8::Number>::return_t)'
   19 |   version->Set(1, Nan::New<v8::Number>(LIBDVBTEE_VERSION_B));
      |                                                            ^
In file included from /home/alarm/.cache/node-gyp/13.12.0/include/node/node.h:67,
                 from ../../nan/nan.h:54,
                 from ../src/node-dvbtee.cc:11:
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
 3547 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note:   candidate expects 3 arguments, 2 provided
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
 3550 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note:   candidate expects 3 arguments, 2 provided
../src/node-dvbtee.cc:20:60: error: no matching function for call to 'v8::Array::Set(int, Nan::imp::FactoryBase<v8::Number>::return_t)'
   20 |   version->Set(2, Nan::New<v8::Number>(LIBDVBTEE_VERSION_C));
      |                                                            ^
In file included from /home/alarm/.cache/node-gyp/13.12.0/include/node/node.h:67,
                 from ../../nan/nan.h:54,
                 from ../src/node-dvbtee.cc:11:
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
 3547 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note:   candidate expects 3 arguments, 2 provided
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
 3550 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note:   candidate expects 3 arguments, 2 provided
In file included from ../../nan/nan.h:54,
                 from ../src/node-dvbtee.cc:11:
../src/node-dvbtee.cc: At global scope:
/home/alarm/.cache/node-gyp/13.12.0/include/node/node.h:618:43: warning: cast between incompatible function types from 'void (*)(Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE)' {aka 'void (*)(v8::Local<v8::Object>)'} to 'node::addon_register_func' {aka 'void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)'} [-Wcast-function-type]
  618 |       (node::addon_register_func) (regfunc),                          \
      |                                           ^
/home/alarm/.cache/node-gyp/13.12.0/include/node/node.h:652:3: note: in expansion of macro 'NODE_MODULE_X'
  652 |   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
      |   ^~~~~~~~~~~~~
../src/node-dvbtee.cc:89:1: note: in expansion of macro 'NODE_MODULE'
   89 | NODE_MODULE(dvbtee, InitAll)
      | ^~~~~~~~~~~
../src/node-dvbtee.cc: In instantiation of 'v8::Local<v8::Array> vectorToV8Array(std::vector<_Tp>) [with T = unsigned char]':
../src/node-dvbtee.cc:51:71:   required from here
../src/node-dvbtee.cc:43:5: error: no matching function for call to 'v8::Array::Set(unsigned int, Nan::imp::IntegerFactory<v8::Int32>::return_t)'
   43 |     a->Set(pos++, Nan::New(*it));
      |     ^
In file included from /home/alarm/.cache/node-gyp/13.12.0/include/node/node.h:67,
                 from ../../nan/nan.h:54,
                 from ../src/node-dvbtee.cc:11:
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
 3547 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3547:37: note:   candidate expects 3 arguments, 2 provided
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
 3550 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
      |                                     ^~~
/home/alarm/.cache/node-gyp/13.12.0/include/node/v8.h:3550:37: note:   candidate expects 3 arguments, 2 provided
make: *** [dvbtee.target.mk:125: Release/obj.target/dvbtee/src/node-dvbtee.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [dvbtee.target.mk:125: Release/obj.target/dvbtee/src/dvbtee-parser.o] Error 1
make: Leaving directory '/home/alarm/local-channels/node_modules/dvbtee/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Linux 5.4.18-udlv7+
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "-j" "8"
gyp ERR! cwd /home/alarm/local-channels/node_modules/dvbtee
gyp ERR! node -v v13.12.0
gyp ERR! node-gyp -v v6.1.0
gyp ERR! not ok 
npm WARN localscraper@1.0.0 No description
npm WARN localscraper@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dvbtee@0.4.4 install: `node-gyp rebuild -j 8`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dvbtee@0.4.4 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/alarm/.npm/_logs/2020-04-12T19_53_44_240Z-debug.log

```

**Comment by mkrufky** [comment]
confirmed.  I'll have a fix pushed up soon.

**Comment by mkrufky** [comment]
@grkblood13  fixed in `dvbtee@0.4.5`
